src/utils/btorutil.c

```diff
@@ -703,3 +704,26 @@ btor_util_getenv_value (BtorMemMgr *mm, const char *lname)
   BTOR_RELEASE_STACK (uname);
   return res;
 }
+
+/*------------------------------------------------------------------------*/
+
+int32_t
+btor_util_timeout_deadline (void *param)
```

Comment:
This function is very specific to the timeout terminate handler. You can also move this as static function to `btorcore.c`.